### PR TITLE
LPS-121078 Adapt tests

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
ci:test:sf

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/layout-data-items/RowWithControls.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ import Row from './Row';
 
 const RowWithControls = React.forwardRef(
 	({children, item, layoutData}, ref) => {
-		const rowConfig = layoutData.items[item.itemId].config;
+		const rowConfig = item.config;
```
Comment (sandrodw3) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/layout-data-items/RowWithControls.js]: Good point 😋 maybe we can use `item.config` directly when calculating rowResponsiveConfig a few lines below

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/layout-data-items/ColumnWithControls.js [hunk 2]
```diff
@@ -76,379 +76,368 @@ export const updateNewLayoutDataContext = (
 	};
 };
 
-const ColumnWithControls = React.forwardRef(
```
Comment (sandrodw3) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/layout-data-items/ColumnWithControls.js:79]: I think you mixed two ideas in the commit message 😛

**Review by sandrodw3 — COMMENTED** [review]
Looks super cool to me 🥳

**Comment by p2kmgcl** [comment]
https://github.com/liferay-echo/liferay-portal/pull/2379

**Pull request closed without merging**